src/components/subHeader/components/description/Description.js

```diff
@@ -0,0 +1,62 @@
+// Libraries
+import PropTypes from 'prop-types'
+
+// components
+import DescriptionBoxList from "../descriptionBoxList";
+import DescriptionAddress from "../descriptionAddress";
+
+// css
+import './description.css'
+
+
+function Description(props){
+
+    const { restaurantInfo, address } = props;
+
+    return (
+        <div className="description inline">
+            <DescriptionAddress address={address} />
```

Comment:
It doesn't only show the address but much more than it, may be you can call it DescriptionHeader